tests/test_basic.py

```diff
@@ -333,7 +333,7 @@ def test():
     client = app.test_client()
     rv = client.get('/')
     assert 'set-cookie' in rv.headers
-    match = re.search(r'\bexpires=([^;]+)(?i)', rv.headers['set-cookie'])
+    match = re.search(r'(?i)\bexpires=([^;]+)', rv.headers['set-cookie'])
```

Comment:
I have no preference, I adapted the inline syntax because that is what was already being used.